src/manager/BaseMapFishPrintManager.ts

```diff
@@ -427,6 +454,7 @@ export class BaseMapFishPrintManager extends Observable {
 
       transform.set('name', BaseMapFishPrintManager.TRANSFORM_INTERACTION_NAME);
 
+      // @ts-ignore
```

Comment:
Really needed?

Reply:
Yes, for some reason the types do not allow `'scaling'` or even `['scaling']` here.